src/applications/mhv/secure-messaging/sass/message-details.scss

```diff
@@ -91,16 +91,24 @@
         height: 20px;
       }
       @media (max-width: $small-screen) {
-        padding-left: 10px;
-        padding-right: 10px;
+        margin-right: 0;
+        margin-left: 0;
+        display: inline-block;
         width: 102%;
       }
     }
     :first-child button {
-      border-radius: 5px 0 0 5px;
+      margin-left: 0;
     }
     :last-child button {
-      border-radius: 0 5px 5px 0;
+      margin-right: 0;

```

Comment:
these :first-child and :last-child selectors were used for handling that border radius on the first and left button. I would guess these are no longer relevant and can be cleared off

Reply:
I am using them to align the first button with the left margin of the other elements on the screen and give enough room for them to spread out and not wrap because of the last button's 10px of margin.